bin/build_container_image

```diff
@@ -2,13 +2,22 @@
 
 REGISTRY=${REGISTRY:-"docker.io"}
 ORGANIZATION=${ORGANIZATION:-"manageiq"}
+
+[[ -z "$TRAVIS_BRANCH" ]] && TRAVIS_BRANCH="$(git rev-parse --abbrev-ref HEAD)"
```

Comment:
Locally, without this, you get a build tagged `latest-`.  After this you get `latest-branch` when not on master.